changelogs/unreleased/gh-6128-fix-crash-with-background-env-option.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Fixed a crash that could happen when Tarantool is started in background mode.
+  For example using environment variable `TT_BACKGROUND=true` (gh-6128).
```

Comment:
Minor suggestions regarding changelogs:
```
* Fixed a crash that could happen when Tarantool is started in the background mode.
  For example, using the environment variable (`TT_BACKGROUND=true`) (gh-6128).
```

Reply:
Andrey, updated, thanks!

Reply:
Applied new suggestions @andreyaksenov shared with me in MyTeam.

> * Fixed a crash that could happen when Tarantool is started in the
> [background mode](https://www.tarantool.io/en/doc/latest/reference/configuration/#confval-background) (gh-6128).